src/config.ts

```diff
@@ -0,0 +1,6 @@
+export default {
```

Comment:
This whole file is somewhat sensitive information and should not be committed, containing this information. What are our options here?